scripts/release.sh

```diff
@@ -30,6 +30,7 @@ Options:
   -d, --dry-run              Output actions that would be taken, but don't run them.
   -h, --help                 Display this text.
   --registry <host[:port]>   Push the built images to the provided registry.
+  --coverage                 Build coverage-enabled version of images where possible.
```

Comment:
Shall we just add the coverage option to debug builds?

Reply:
The first requirement is for coverage builds matching release code.
That is what this change is for.